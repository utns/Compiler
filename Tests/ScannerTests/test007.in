(12*0.454532-213)/989.00001