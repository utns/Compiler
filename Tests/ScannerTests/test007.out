1    1   delimiter       (                    (
1    2   integer number  12                   12
1    4   operation       *                    *
1    5   real number     0.454532             0.454532
1    13  operation       -                    -
1    14  integer number  213                  213
1    17  delimiter       )                    )
1    18  operation       /                    /
1    19  real number     989.00001            989.000010
